Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -286,6 +287,24 @@ open class AVFoundationPlayback: Playback {
             selector: #selector(AVFoundationPlayback.playbackDidEnd),
             name: NSNotification.Name.AVPlayerItemDidPlayToEndTime,
             object: player.currentItem)
+        
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(AVFoundationPlayback.onAccessLogEntry),
+            name: NSNotification.Name.AVPlayerItemNewAccessLogEntry,
+            object: nil)
+    }
+    
+    @objc func onAccessLogEntry(notification: NSNotification?) {
```

Comment:
I think we can create an extension of AVPlayerItem to encapsulate this code:
```swift
extension AVPlayerItem {
    var bitrate: Double? {
        return accessLog()?.events.last?.bitrate ?? 0.0
    }
}
```

ending up with a simpler guard let statement:
```swift
    @objc func onAccessLogEntry(notification: NSNotification?) {
        guard let playerItem = notification?.object as? AVPlayerItem,
            lastBitrate != playerItem.bitrate else { return }

        lastBitrate = playerItem.bitrate
        trigger(.didUpdateBitrate, userInfo: ["bitrate": lastBitrate])
    }
```